### PR TITLE
Table: expand row feature only supports scoped slot.

### PR DESCRIPTION
**Pull request by QingWei-Li**
Please makes sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's [Contributing Guide](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.md).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

File: examples/docs/en-US/table.md [hunk 1]
```diff
@@ -1349,19 +1349,19 @@ Customize table column so it can be integrated with other components.
 ### Expandable row
 
 When the row content is too long and you do not want to display the horizontal scroll bar, you can use the expandable row feature.
-:::demo Activate expandable row by adding type="expand" and `inline-template` attribute，The template for `el-table-column` will be rendered as the contents of the expanded row, you can access the same attributes as the` inline-template`。
+:::demo Activate expandable row by adding type="expand" and `Scoped slot`, The template for `el-table-column` will be rendered as the contents of the expanded row, you can access the same attributes as the` Scoped slot`.
```
Comment (Leopoldthecoder) [inline examples/docs/en-US/table.md]: Activate expandable row by adding type="expand" and `scoped slot`. The template for `el-table-column` will be rendered as the contents of the expanded row, and you can access the same attributes as when you are using`scoped slot` in custom column templates.

File: examples/docs/zh-CN/table.md [hunk 2]
```diff
@@ -1363,19 +1363,19 @@
 ### 展开行
 
 当行内容过多并且不想显示横向滚动条时，可以使用 Table 展开行功能。
-:::demo 通过设置 type="expand" 和 `inline-template` 属性可以开启展开行功能，`el-table-column` 的模板会被渲染成为展开行的内容，展开行可访问的属性与使用 `inline-template` 的时候相同。
+:::demo 通过设置 type="expand" 和 `Scoped slot` 可以开启展开行功能，`el-table-column` 的模板会被渲染成为展开行的内容，展开行可访问的属性与使用 `Scoped slot` 的时候相同。
```
Comment (Leopoldthecoder) [inline examples/docs/zh-CN/table.md]:
```
展开行可访问的属性与使用 `Scoped slot` 的时候相同。
```
是不是改成「展开行可访问的属性与使用自定义列模板时的`scoped slot`相同」

File: examples/docs/en-US/table.md [hunk 3]
```diff
@@ -1349,19 +1349,19 @@ Customize table column so it can be integrated with other components.
 ### Expandable row
 
 When the row content is too long and you do not want to display the horizontal scroll bar, you can use the expandable row feature.
-:::demo Activate expandable row by adding type="expand" and `inline-template` attribute，The template for `el-table-column` will be rendered as the contents of the expanded row, you can access the same attributes as the` inline-template`。
+:::demo Activate expandable row by adding type="expand" and scoped slot. The template for el-table-column will be rendered as the contents of the expanded row, and you can access the same attributes as when you are usingscoped slot in custom column templates.
```
Comment (Leopoldthecoder) [inline examples/docs/en-US/table.md]: usingscoped => using scoped

Comment (Leopoldthecoder) [inline examples/docs/en-US/table.md]: 哦，直接点编辑按钮然后再复制吧

**Comment by LiuGabi** [comment]
I hope to supports more.

**Comment by ranransun** [comment]
请问下怎么解决的这个问题啊 加动画效果

**Pull request merged**